data/transactions/logic/debugger_test.go

```diff
@@ -174,3 +174,27 @@ func TestValueDeltaToValueDelta(t *testing.T) {
 	require.Equal(t, base64.StdEncoding.EncodeToString([]byte(vDelta.Bytes)), ans.Bytes)
 	require.Equal(t, vDelta.Uint, ans.Uint)
 }
+
+func TestParseCallstack(t *testing.T) {
```

Comment:
add a real test for with two nested callstack received from the evaluator via debugger.Update.

Reply:
Good point - added unit test below (real test checks `line+1` due to `pragma`)